test/scriptFlywheel/unit/withdrawTests.spec.ts

```diff
@@ -677,6 +677,81 @@ contract("SavingAccount.withdraw", async (accounts) => {
+
+                    it("Withdraw accrued COMP tokens", async function () {
+                        this.timeout(0);
+
+                        // Deploy mock ERC20 token for testing
+                        const name = "Test Token";
+                        const symbol = "TTKN";
+                        const decimals = new BN(18);
+                        const initialSupply = new BN(10000);
+
+                        const erc20TTKN: t.MockErc20Instance = await ERC20.new(
+                            name,
+                            symbol,
+                            decimals,
+                            initialSupply
+                        );
+
+                        // Transfer mock tokens to SavingAccount
+                        await erc20TTKN.transfer(
+                            savingAccount.address,
+                            initialSupply.mul(eighteenPrecision)
+                        );
+
+                        const savingAccountBalTTKN = new BN(
+                            await erc20TTKN.balanceOf(savingAccount.address)
+                        );
+                        const depositAmount = new BN(1000);
+                        await erc20DAI.approve(savingAccount.address, new BN(1500));
+
+                        let userBalanceBeforeWithdrawTTKN = await erc20TTKN.balanceOf(owner);
+                        let accountBalanceBeforeWithdrawDAI = await erc20DAI.balanceOf(
+                            savingAccount.address
+                        );
+                        const balSavingAccountUserBefore = await erc20DAI.balanceOf(
+                            savingAccount.address
+                        );
+                        const balCTokensBefore = new BN(
+                            await cDAI.balanceOfUnderlying.call(savingAccount.address)
+                        );
+
+                        // deposit tokens
+                        await savingAccount.deposit(erc20DAI.address, depositAmount);
+
+                        const savingAccountCDAITokenAfterDeposit = BN(
+                            await cDAI.balanceOfUnderlying.call(savingAccount.address)
+                        );
+                        const savingAccountDAITokenAfterDeposit = BN(
+                            await erc20DAI.balanceOf(savingAccount.address)
+                        );
+
+                        await savAccBalVerify(
+                            0,
+                            depositAmount,
+                            erc20DAI.address,
+                            cDAI,
+                            balCTokensBefore,
+                            BN(balSavingAccountUserBefore),
+                            bank,
+                            savingAccount
+                        );
+
+                        await savingAccount.fastForward(10000);
+                        // deposit for rate checkpoint
+                        await savingAccount.deposit(erc20DAI.address, new BN(10));
```

Comment:
Why a user need to deposit some times to test only `withdrawCOMP()` ?

Reply:
That was from a previous logic I was thinking of, similar to the accrual on Compound. 
But I don't think we need deposit now, so I'll remove it